### ISSUE TITLE
Errors in version 1.5

### ISSUE THREAD

**Issue by krabina** [ProfessionalWiki/Network#26]
When upgrading from 1.4 to 1.5, network examples that already worked are not working anymore. Just a white space is shown. These are the error messages:

MediaWiki 1.31:
```
Die Ressource von "/Vendor/onoi/shared-resources/res/jquery.qtip/extended/jquery.qtip.css" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Die Ressource von "/Vendor/onoi/shared-resources/res/localForage/localforage.min.js" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Laden fehlgeschlagen für das <script> mit der Quelle "/vendor/onoi/shared-resources/res/localForage/localforage.min.js?89a55". Network:1:1
Die Ressource von "/Vendor/onoi/shared-resources/res/md5/jquery.md5.js" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Laden fehlgeschlagen für das <script> mit der Quelle "/vendor/onoi/shared-resources/res/md5/jquery.md5.js?32266". Network:1:1
Die Ressource von "/Vendor/onoi/shared-resources/res/onoi.util.css" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Die Ressource von "/Vendor/onoi/shared-resources/res/jquery.qtip/extended/jquery.qtip.js" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Laden fehlgeschlagen für das <script> mit der Quelle "/vendor/onoi/shared-resources/res/jquery.qtip/extended/jquery.qtip.js?18428". Network:1:1
Die Ressource von "/Vendor/onoi/shared-resources/res/onoi.blobstore.js" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Laden fehlgeschlagen für das <script> mit der Quelle "/vendor/onoi/shared-resources/res/onoi.blobstore.js?6f877". Network:1:1
Die Ressource von "/Vendor/onoi/shared-resources/res/onoi.util.js" wurde wegen eines MIME-Typ-Konfliktes ("text/html") blockiert (X-Content-Type-Options: nosniff).
Network
Laden fehlgeschlagen für das <script> mit der Quelle "/vendor/onoi/shared-resources/res/onoi.util.js?44ee1". Network:1:1
Uncaught ReferenceError: onoi is not defined
    <anonymous> /extensions/SemanticCite/res/scite.tooltip.js?ecade:153
scite.tooltip.js:153:1
Uncaught TypeError: pageInfoResponse.query.pages.forEach is not a function
    _apiResponseToPagesAndLinks /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:51
    jQuery 3
    xhr /resources/src/mediawiki/api.js?8308a:274
    jQuery 4
ApiPageConnectionRepo.js:51:35
```

MW 1.35:
```

`Source-Map-Fehler: Error: request failed with status 404
Ressourcen-Adresse: /extensions/Network/resources/lib/vis-network.js?6101f
Source-Map-Adresse: vis-network.js.map

Uncaught TypeError: pageInfoResponse.query.pages.forEach is not a function
    _apiResponseToPagesAndLinks /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:51
    jQuery 3
    xhr /resources/src/mediawiki.api/index.js?06bae:273
    jQuery 7
    ajax /resources/src/mediawiki.api/index.js?06bae:245
    get /resources/src/mediawiki.api/index.js?06bae:137
    _queryPageNodeInfo /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:97
    _apiResponseToPagesAndLinks /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:45
    _apiResponseToPagesAndLinks/extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:41
    addConnections /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:32
    jQuery 3
    xhr /resources/src/mediawiki.api/index.js?06bae:273
    jQuery 7
    ajax /resources/src/mediawiki.api/index.js?06bae:245
    get /resources/src/mediawiki.api/index.js?06bae:137
    _queryLinks /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:73
    addConnections /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:30
    addConnections /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:21
    showPages /extensions/Network/resources/js/Network.js?044f1:31
    <anonymous> /extensions/Network/resources/js/index.js?6666f:21
    jQuery 2
    <anonymous> /extensions/Network/resources/js/index.js?6666f:6
    jQuery 2
    <anonymous> /extensions/Network/resources/js/index.js?6666f:5
    <anonymous> u/extensions/Network/resources/js/index.js?6666f:34`
```

Both wikis are running PHP 7.3.27 (fpm-fcgi) and MariaDB 10.3.29

**Comment by kghbln** [comment]
Thanks for reporting. Version 1.5.0 was not tagged and released yet. Probably you are running master. Anyhow good to know. @JeroenDeDauw FYI

**Comment by krabina** [comment]
ah, yes, sorry. I am running master which displays a 1.5.0 in Special:Version

**Comment by krabina** [comment]
Ok. For reference: both instances use the latest SMW version (3.2.3)

**Comment by cicalese** [comment]
> Uncaught TypeError: pageInfoResponse.query.pages.forEach is not a function
>    _apiResponseToPagesAndLinks /extensions/Network/resources/js/ApiPageConnectionRepo.js?9c5ff:51

This was just fixed in #25.

**Comment by kghbln** [comment]
Really cool.
